pkg/ebpf/c/include/tracee/arch.h

```diff
@@ -37,24 +47,26 @@ static __always_inline bool is_compat(struct task_struct *task)
     return is_x86_compat(task);
 #elif defined(bpf_target_arm64)
     return is_arm64_compat(task);
-#else
-    return false;
 #endif
 }
 
+// Pick syscall IDs using registers
+
 static __always_inline int get_syscall_id_from_regs(struct pt_regs *regs)
 {
 #if defined(bpf_target_x86)
     int id = READ_KERN(regs->orig_ax);
 #elif defined(bpf_target_arm64)
     int id = READ_KERN(regs->syscallno);
 #endif
+
     return id;
 }
 
+// Pick registers from task stack (tested with 4k pages only)
```

Comment:
The comment should say that THREAD_SIZE is defined for 4k pages only, and not that we tested for 4k pages only

Reply:
> I disagree with some of these renamings (e.g. common->include and splitting kernel and tracee), yet I think we should merge the other commits.

kernel = helpers to access kernel types and members
tracee = prototypes, helpers, definitions, constants internal to tracee

But this was just a mid step until doing it correctly (and starting to build different objects). I have considered it for this milestone because its also a fix (headers inclusion now works by accident as many of them are "" and others are <> and it totally depends on dir include ordering and things like that).

Im ok in moving to the next milestone.

Reply:
All what you call "kernel" accessors were supposed to be under one file in the original split suggestion, so I agree it should be grouped. Let's not do it in this PR though, I think we need to discuss the file hierarchy first